README.md

```diff
@@ -77,8 +77,9 @@ pager resolve <incident ID>        - Resolve a specific incident
 ### Schedules
 
 ```
-pager oncall - List available schedules
-pager oncall <schedule> - Show who is on call for the given schedule
+pager oncall                       - List available schedules
+pager oncall <schedule>            - Show who is on call for the given schedule
+pager base <schedule>              - Show the base layer of a schedule. This is useful if your schedule has a foundation rotation (eg weekly), with layers on top.
```

Comment:
I tested locally, looks good – just one thought on the new `base` command: could be nice to have it return an error if someone does `pager base` without specifying a schedule, since right now it silently returns and doesn't indicate user error

Reply:
That's a good idea. I can't remember why I didn't do that, but I will.

Reply:
I've gone one step further and use the same behavior as `oncall`, which lists the possible schedules if none are provided.